server/src/main/java/org/elasticsearch/common/io/stream/StreamInput.java

```diff
@@ -829,6 +829,18 @@ private Map<String, Object> readLinkedHashMap() throws IOException {
         return map9;
     }
 
+    private Map<Object, Object> readObjectKeyLinkedHashMap() throws IOException {
+        int size9 = readArraySize();
```

Comment:
Maybe just `size` and `map`? I'm not sure where the numbers came from.

Reply:
The numbers come, I think, from the fact that 9 or 10 is the identifier used to serialize and deserialize the map distinguishing the two. I just kept the same naming used in the other two methods for "consistency".

Reply:
Ah. Well, it doesn't really matter anyway if we don't use a new method.